ReText/table.py

```diff
@@ -0,0 +1,91 @@
+from ReText import globalSettings
+from PyQt5.QtCore import pyqtSignal, Qt
+from PyQt5.QtWidgets import QDialog, QVBoxLayout, QTabWidget, \
+	QDialogButtonBox, QWidget, QGridLayout, QLabel, QSpinBox
```

Comment:
In new files please use 4 spaces indentation. Old files have indentation by tabs but that is legacy and I want to remove that in the future.

Reply:
I modified to use 4 spaces indentation instead of Tab!